.circleci/config.yml

```diff
@@ -29,6 +29,22 @@ jobs:
           root: .
           paths: site
 
+  build_docs_pdf:
+    working_directory: ~/bids-specification/pdf_build_src
+    docker:
+      - image: danteev/texlive:TL2017
+    steps:
+      - checkout:
+          path: ~/bids-specification
+      - run: 
+          name: generate pdf version docs 
+          command: sh build_pdf.sh
+      - store_artifacts:
+          path: bids-spec.pdf
+      - run:
+          name: remove pdf version from repo
+          command: rm bids-spec.pdf
```

Comment:
Checking the CircleCI build (e.g., https://circleci.com/gh/bids-standard/bids-specification/1880?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) I see that the pdf gets built and the artifact is uploaded ...

However, I do not see the "artifact tab" and thus cannot inspect the PDF. 

I wonder whether that's because of this last part: 

```
 - run:
          name: remove pdf version from repo
          command: rm bids-spec.pdf
```

Could you simply remove that? The pdf should not be committed to the repository anyhow, so the removal step should be unnecessary (and might be detrimental with regards to the artifact)

Reply:
EDIT: I just had to re-log into the CircleCI interface ... I do see the artifact now!

https://1880-150465237-gh.circle-artifacts.com/0/bids-spec.pdf

awesome job! :-)

Reply:
Great! Do you still think it's necessary to remove the quoted lines of code in the config file?

Reply:
I think it doesn't matter. But since everything is working now, let's just leave it as is for now.